.github/workflows/unticketed-pr.yml

```diff
@@ -0,0 +1,28 @@
+name: Add unticketed PRs to github board
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  add_issue:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PR: ${{ toJSON(github.event.pull_request) }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.18.0"
+
+      - name: Verify PR
+        run: go run testutil/verifypr/verifypr.go
+
+      - name: Set env.NONE_TICKET_PRESENT
+        run: go run testutil/unticketedpr/unticketedpr.go
+
+      - name: Add PR To Project
+        if: ${{ success() && env.NONE_TICKET_PRESENT }}
```

Comment:
see issue, but you need to add size and sprint and status, not sure this action supports that, you need to manually add it to the project from `trackpr`

Reply:
yeah, this github action is pretty basic and just adds to project board, can't assign size, sprint etc. will do it manually then